.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: go
 
 go:
-  - 1.11.x
```

Comment:
Nope, we support the last two versions of Go. You'll need to make some version-tagged files for this PR.

Reply:
the last two versions of Go is now 1.14.x and 1.13.x as of 23 Nov 2020. Should we go with this update now?

Reply:
Sure. Open up a new PR that is mergeable and I'll merge it. Be sure to @ me or I won't see it.